app/cafe.py

```diff
@@ -0,0 +1,26 @@
+import datetime
+
+
+from app.errors import (
+    NotVaccinatedError,
+    NotWearingMaskError,
+    OutdatedVaccineError
+)
+
+
+class Cafe:
+
+    def __init__(self, name: str) -> None:
+        self.name = name
+
+    def visit_cafe(self, visitor: dict) -> str:
+        if "vaccine" not in visitor:
+            raise NotVaccinatedError(f"{visitor['name']} must be vaccinated!")
+
+        if visitor["vaccine"]["expiration_date"] < datetime.date.today():
+            raise OutdatedVaccineError("Your vaccine has expired!")
+
+        if visitor["wearing_a_mask"] is False:

```

Comment:
use .get("wearing_a_mask") possible wearing_a_mask can not exist in visitor. Also you can write as:
```suggestion
        if not visitor.get("wearing_a_mask"):

```